### DISCUSSION TITLE
[DOUBT]  Why is this step optional ? ->  Enable BTF BPF Type Format (BTF) (Optional, 2022-04-17)

### DISCUSSION THREAD

**Discussion by Greatz08** [gojue/ecapture#620] [General]
Is it not required or do we have inbuilt support for it in latest kernels ?

**Comment by cfc4n** [comment]
If it's an old kernel (below version 5.0), this configuration is typically disabled by default in many Linux distributions and requires compiling with the `make nocore` command. In newer kernels, this feature is enabled by default, so you can compile using just the `make` command. 

Therefore, for new kernels, there's no need to pay attention to this statement.

**Reply by mbbsadmission20** [reply]
CBS Innovations Pvt Ltd is a dynamic leader in the field of sustainable
heating and lighting solutions. With a commitment to innovation and environmental responsibility, we have quickly become a trusted provider of comprehensive heating and lighting solutions for residences, hotels, and corporate clients
PELLET BURNER
PELLET BASED ROOM STOVE
FROZEN PIPE SOLUTION
ELECTRIC HAMAM
SOLAR SOLUTIONS

https://cbsgreen.in/
https://cbsgreen.in/pellet-burner-product
https://cbsgreen.in/electric-hamam
https://cbsgreen.in/solar-solution
https://cbsgreen.in/pellet-based-room-stove-product
https://cbsgreen.in/frozen-pipes-solution
https://cbsgreen.in/contact-cbs

<a href="https://cbsgreen.in/">CBS Innovations</a>
<a href="https://cbsgreen.in/">Electric Hamam</a>
<a href="https://cbsgreen.in/solar-solution/">Solar</a>
<a href="https://cbsgreen.in/">Pellet Burner</a>
<a href="https://cbsgreen.in/">Pellet based room stove</a>
